### ISSUE TITLE
Katowice

### ISSUE THREAD

**Issue by deusmaks** [kgns/weapons#77]
Hi. You are planning add skins Katowice case?

**Comment by kgns** [comment]
what katowice case?

**Comment by deusmaks** [comment]
This Katowice Case. New case and new skins weapons. Watch - https://www.youtube.com/watch?v=PRR0ScxMRpY

**Comment by EDUAaron** [comment]
> s

It's the Control panel and shit like that, you've had it for ages

**Comment by kgns** [comment]
ok let me rephrase, can you send a link to the new skins added with katowice cases? I don't have time to research, feed me with data, I'll take a look afterwards

**Comment by ThanhByKyTa** [comment]
@kgns update Prisma Case

**Comment by kgns** [comment]
there are no new skins from Katowice 2019. I think you guys are referring to
https://csgostash.com/collection/The+2018+Nuke+Collection
it is from London 2018, and are already in the plugin
